src/Character.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Character = (img, name, species) => (
```

Comment:
think you need to deconstruct these.
const Character = ({ img, name, species }) => (